REQUIRE

```diff
@@ -1,3 +1,4 @@
 julia 0.3
 Polynomials
 Compat 0.4.1
+ProgressMeter
```

Comment:
Yes, Chris is right.  One PR one topic.  So PrgressMeter belongs to a different PR.